helpers/ndk.psm1

```diff
@@ -349,7 +347,6 @@ function Invoke-NDKPerfNtoN {
         }
     }
 
-
     $ServerBytesPerSecond = 0
     $ServerBpsArray = @()

```

Comment:
should these be removed as well?

Reply:
I believe those are part of Stage 6 but I am similarly taking a look at the counter collection for that for the next PR and will remove if needed!